test/lib/jdk/test/lib/compiler/InMemoryJavaCompiler.java

```diff
@@ -162,6 +169,86 @@ public boolean hasLocation(Location location) {
 
     }
 
+    // Wraper for class file
+    static class ClassFile extends SimpleJavaFileObject {
```

Comment:
The original class has 
MemoryJavaFileObject
already which could be removed now once we have 
ClassFile and SourceFile